engine/src/ast/function_expr.rs

```diff
@@ -23,13 +23,15 @@ impl<'s> FunctionCallArgExpr<'s> {
         }
     }
 
-    pub fn execute(&'s self, ctx: &'s ExecutionContext<'s>) -> LhsValue<'s> {
+    pub fn execute(&'s self, ctx: &'s ExecutionContext<'s>) -> Option<LhsValue<'s>> {
```

Comment:
Could introduce an execute_opt, so original maintains panic behavior, and execute_opt returns option.